pynecone/components/base/link.py

```diff
@@ -19,3 +21,33 @@ def _render(self) -> Tag:
             href=self.href,
             rel=self.rel,
         )
+
+class ScriptTag(Component):
+    """A component that creates a script tag with the speacified type and source."""
+
+    type: Var[str]
+
+    source: Var[str]
+
+    integrity: Optional[Var[str]] = None
+
+    crossorigin: Optional[Var[str]] = None
+
+    referrer_policy: Optional[Var[str]] = None
+
+    is_async: Optional[Var[bool]] = None
+    
+    defer: Optional[Var[bool]] = None
+
+
+
+    def _render(self) -> Tag:
```

Comment:
Once you add the `tag = "script"` prop for the component, we can get rid of this function I believe. The `super()._render` method should handle this.